bootstrap/cmd/bootstrap/app/ksServer.go

```diff
@@ -1234,12 +1259,20 @@ func makeDeployEndpoint(svc KsService) endpoint.Endpoint {
 			return r, err
 		}
 
-		DmDeployment, err := svc.InsertDeployment(ctx, req)
+		if req.CreatePersistentStorage {
+			_, err := svc.InsertDeployment(ctx, req, StorageDmSpec)
```

Comment:
Can we check if the deployment of storage actually succeeded?

Reply:
yeah i've verified and saw it created ok

Reply:
I mean in "finishDeployment" can we check if both InsertDeployment succeeded?

Reply:
done